### PR TITLE
Secneur x adding output for error msg

### PR DESCRIPTION
**Pull request by karinafishman**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-7850

## Description
- Added the error message to Playbook's output.
- Added task for verifying if the file type supported by the Vendor 

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/JoeSecurity/Playbooks/playbook-Detonate_File_-_JoeSecurity_V2_README.md [hunk 1]
```diff
@@ -30,10 +30,9 @@ This playbook does not use any sub-playbooks.
...
 | Interval | Duration for executing the pooling \(in minutes\) | 1 | Optional |
-| Timeout | The duration after which to stop pooling and to resume the playbook \(in minutes\) | 15 | Optional |
-| Systems | Operating system to run the analysis on \(comma-separated\). Supported values are: w7, w7x64, w7_1, w7_2, w7native, android2, android3, mac1, w7l, w7x64l, w10, android4, w7x64native, w7_3, w10native, android5native_1, w7_4, w7_5, w10x64, w7x64_hvm, android6, iphone1, w7_sec, macvm, w7_lang_packs, w7x64native_hvm, lnxubuntu1, lnxcentos1, android7_nougat |  | Optional |
+| Timeout | The duration after which to stop pooling and to resume the playbook \(in minutes\) | 1500 | Optional |
+| Systems | Operating system to run the analysis on \(comma-separated\). Supported values are: w7, w7x64, w7_1, w7_2, w7native, android2, android3, mac1, w7l, w7x64l, w10, android4, w7x64native, w7_3, w10native, android5native_1, w7_4, w7_5, w10x64, w7x64_hvm, android6, iphone1, w7_sec, macvm, w7_lang_packs, w7x64native_hvm, lnxubuntu1, lnxcentos1, android7_nougat \(if no input is provided, the default is w10x64_office\) |  | Optional |
```
Comment (ShirleyDenkberg) [inline Packs/JoeSecurity/Playbooks/playbook-Detonate_File_-_JoeSecurity_V2_README.md:34]:
```suggestion
| Systems | Operating system to run the analysis on \(comma-separated\). Supported values are: w7, w7x64, w7_1, w7_2, w7native, android2, android3, mac1, w7l, w7x64l, w10, android4, w7x64native, w7_3, w10native, android5native_1, w7_4, w7_5, w10x64, w7x64_hvm, android6, iphone1, w7_sec, macvm, w7_lang_packs, w7x64native_hvm, lnxubuntu1, lnxcentos1, android7_nougat. \(If no input is provided, the default is w10x64_office.\) |  | Optional |
```

File: Packs/SecneurXAnalysis/Playbooks/Detonate_File_-_SecneurX_Analysis_README.md [hunk 2]
```diff
@@ -53,8 +59,10 @@ This playbook does not use any scripts.
 | SecneurXAnalysis.Report.FileDeleted | List of File deletions observed in the analyzed sample | string |
 | SecneurXAnalysis.Report.FileModified | List of File changes observed in the analyzed sample | string |
 | SecneurXAnalysis.Report.IOC | List of IOC's observed in the analyzed sample | string |
-| SecneurXAnalysis.Report.Status | Analysis queued sample state | string |
+| SecneurXAnalysis.Report.Status | Analysis queued sample state | String |
```
Comment (ShirleyDenkberg) [inline Packs/SecneurXAnalysis/Playbooks/Detonate_File_-_SecneurX_Analysis_README.md:62]:
```suggestion
| SecneurXAnalysis.Report.Status | Analysis queued sample state. | String |
```
Why is this "String" type uppercase, when all the other "string" types are lower case?

**Comment by ShirleyDenkberg** [comment]
@karinafishman Need content in the Release Notes.
@ArikDay Doc review completed.

File: Packs/SecneurXAnalysis/doc_files/Detonate_File_-_SecneurX_Analysis.png
Comment (ArikDay) [inline Packs/SecneurXAnalysis/doc_files/Detonate_File_-_SecneurX_Analysis.png:1]: Screenshots should not be in dark mode

File: Packs/JoeSecurity/doc_files/Detonate_File_-_JoeSecurity_V2.png
Comment (ArikDay) [inline Packs/JoeSecurity/doc_files/Detonate_File_-_JoeSecurity_V2.png:1]: Screenshots should not be in dark mode

File: Packs/SecneurXAnalysis/ReleaseNotes/1_0_7.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+
+#### Playbooks
+
+##### Detonate File - SecneurX Analysis
+
+- Added new image of the playbook
```
Comment (ArikDay) [inline Packs/SecneurXAnalysis/ReleaseNotes/1_0_7.md:6]: Added a playbook image to the pack readme

File: Packs/SecneurXAnalysis/ReleaseNotes/1_0_7.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+- Added task for checking supported file types
```
Comment (ArikDay) [inline Packs/SecneurXAnalysis/ReleaseNotes/1_0_7.md:7]: Added conditional task for checking supported file types

File: Packs/SecneurXAnalysis/ReleaseNotes/1_0_7.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+- Added error message to the playbook output
```
Comment (ArikDay) [inline Packs/SecneurXAnalysis/ReleaseNotes/1_0_7.md:8]: Please provide more details. When does this error message occur?

File: Packs/SecneurXAnalysis/Playbooks/Detonate_File_-_SecneurX_Analysis.yml [hunk 6]
```diff
@@ -379,9 +442,9 @@ view: |-
...
       "dimensions": {
-        "height": 1455,
-        "width": 740,
-        "x": 1110,
+        "height": 1545,
+        "width": 1230,
+        "x": 1450,
         "y": -20
       }
     }
```
Comment (ArikDay) [inline Packs/SecneurXAnalysis/Playbooks/Detonate_File_-_SecneurX_Analysis.yml:450]: It doesn't looks like you added the error_msg output to the playbook in this PR.

File: Packs/JoeSecurity/ReleaseNotes/1_1_16.md [hunk 7]
```diff
@@ -0,0 +1,7 @@
+
+#### Playbooks
+
+##### Detonate File - JoeSecurity V2
+
+- Added a task to verify that the uploaded files are supported by the vendor.
```
Comment (ArikDay) [inline Packs/JoeSecurity/ReleaseNotes/1_1_16.md:6]: File types? if so, please add

File: Packs/JoeSecurity/Playbooks/playbook-Detonate_File_-_JoeSecurity_V2.yml [hunk 8]
```diff
@@ -335,8 +337,52 @@ tasks:
...
+      brand: ""
+      description: ''
+    nexttasks:
+      '#default#':
+      - "5"
+      "yes":
+      - "1"
+    separatecontext: false
+    conditions:
+    - label: "yes"
```
Comment (ArikDay) [inline Packs/JoeSecurity/Playbooks/playbook-Detonate_File_-_JoeSecurity_V2.yml:370]: Should it be with ignore case?

File: Packs/SecneurXAnalysis/ReleaseNotes/1_0_7.md [hunk 9]
```diff
@@ -0,0 +1,8 @@
+
+#### Playbooks
+
+##### Detonate File - SecneurX Analysis
+
+- Added a playbook image to the playbook readme
+- Added conditional task for checking supported file types.
+- Added error message to the playbook output when when the file type is not supported.
```
Comment (ArikDay) [inline Packs/SecneurXAnalysis/ReleaseNotes/1_0_7.md:8]: "when" twice

File: Packs/JoeSecurity/ReleaseNotes/1_1_16.md [hunk 10]
```diff
@@ -0,0 +1,7 @@
+
+#### Playbooks
+
+##### Detonate File - JoeSecurity V2
+
+- Added a task to verify that the uploaded files types are supported by the vendor.
```
Comment (ArikDay) [inline Packs/JoeSecurity/ReleaseNotes/1_1_16.md:6]: "file types" not "files types"

**Review by ArikDay — APPROVED** [review]

**Pull request merged**